tonic/handler.go

```diff
@@ -192,6 +192,14 @@ func bind(c *gin.Context, v reflect.Value, tag string, extract extractor) error
 		if tagValue == "" {
 			continue
 		}
+		// Set-up context for extractors.
+		// Query.
+		explode, ok := ft.Tag.Lookup(ExplodeTag)
+		if ok && explode == "false" {
```

Comment:
ParseBool ?

Reply:
I'd rather limit the scope of the accepted value here to only `false`. `strconv.ParseBool` accept `1, t, T, TRUE, true, True, 0, f, F, FALSE` and return an error for any other input, which again, I'd prefer not have to deal with.

Reply:
I understand, though I would think sticking with this widely used function is more in line with assumptions/expectations when discovering the feature, which would make it more intuitive, despite some of these values looking weird.

Reply:
if err != nil, stick to the default (true) ?

Reply:
Good for me, amending.